code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -547,6 +547,9 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 				if (other_ruleset.flags & HIGH_IMPACT_RULESET)
 					drafted_rules[other_ruleset] = null
 
+		if (ruleset.flags & NO_OTHER_ROUNDSTARTS_RULESET)
+			drafted_rules.Cut()
```

Comment:
I can't say what, but something about me just doesn't trust this line of code.

Reply:
Trust me, this is how it works. I know it's weird, I agree, but setting a ruleset to null in drafted_rules prevents it from being executed (see the high impact above).

The current ruleset still gets added.

Reply:
Ah, there is one problem here, it won't work if another ruleset has been picked already.